### ISSUE TITLE
Obsolete GO:0008313 gurken-activated receptor activity and def change to GO:0005006 epidermal growth factor-activated receptor activity

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-ontology#23056]
Term:GO:0008313 gurken-activated receptor activity

This has never been used - refers the ligand rather than receptor - think we would use epidermal growth factor-activated receptor activity in preference

Could we also update the term GO:0005006 epidermal growth factor-activated receptor activity to reflect the receptor rather than the ligand?
From
"Combining with an epidermal growth factor and transmitting the signal across the plasma membrane to initiate a change in cell activity. "
to
"Combining with an epidermal growth factor receptor ligand and transmitting the signal across the plasma membrane to initiate a change in cell activity. "

This gets rid of the need to have the ligand to have an EGF fold and therefore becomes sequence agonistic - the ligand can be captured by "has_input"

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete 'GO:0008313 gurken-activated receptor activity'. The reason for obsoletion is that this refers to the ligand, not the receptor. There are no annotations to this term, these term is not present in any subset. 

You can comment on the ticket: 

https://github.com/geneontology/go-ontology/issues/23056

Thanks, Pascale

**Comment by pgaudet** [comment]
@hattrill We also have the following terms build on the same pattern: 

* GO:0042813 Wnt-activated receptor activity
* GO:0005009 insulin-activated receptor activity 
* GO:0005010 insulin-like growth factor-activated receptor activity
* GO:0015057 thrombin-activated receptor activity
* GO:0005006 epidermal growth factor-activated receptor activity
* GO:0036332 placental growth factor-activated receptor activity
* GO:0005007 fibroblast growth factor-activated receptor activity
* GO:0005008 hepatocyte growth factor-activated receptor activity
* GO:0005031 tumor necrosis factor-activated receptor activity
I could change the labels to remove the 'activated', this way we refer to the receptor - would that work? 

Thanks, Pascale

**Comment by hattrill** [comment]
Yes, that sounds like a good idea. Placing the 'activity' on the receptor's properties rather than the agonist.
